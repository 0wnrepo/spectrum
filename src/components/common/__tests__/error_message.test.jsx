import React from 'react';
import renderer from 'react-test-renderer';

import ErrorMessage from '../error_message';

it('renders correctly', () => {
  const tree = renderer.create(<ErrorMessage content="test" />).toJSON();
  expect(tree).toMatchSnapshot();
});
