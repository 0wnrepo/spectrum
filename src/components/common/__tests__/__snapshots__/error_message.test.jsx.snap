// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorMessage /> renders correctly 1`] = `
<Message
  error={true}
  icon={true}
>
  <Icon
    as="i"
    name="warning sign"
  />
  <MessageContent>
    <MessageHeader>
      Oops, something went wrong
    </MessageHeader>
    Test Error message
  </MessageContent>
</Message>
`;
